templates/index.html

```diff
@@ -0,0 +1,111 @@
+<html>
+
+<head>
+    <title>Monitoring Accessible Charts</title>
+    <script src="https://d3js.org/d3.v5.min.js"></script>
+</head>
+
+<body>
+    <h2>Monitoring Accessible Charts</h2>
+
+    <div>
+        <select name="selectedMode" id="modeSelector"></select>
+    </div>
+    <svg width="600" height="400" id="chart"></svg>
+    
+    <script>
```

Comment:
Let's refactor this into a separate JavaScript file. As the frontend becomes more complicated, inlining scripts like this won't be sustainable. Using separate JavaScript files also means that we can test your code programatically.